src/main/java/org/spongepowered/api/registry/RegistryHolder.java

```diff
@@ -57,6 +57,13 @@ public interface RegistryHolder {
      */
     <T> Optional<Registry<T>> findRegistry(final RegistryType<T> type);
 
+    /**
+     * Gets a {@link Stream} of the {@link Registry registries} in this holder.
+     *
+     * @return The stream
+     */
+    Stream<Registry<?>> streamRegistries();
```

Comment:
Maybe just `registries`? Rename the below to that but has the parameter

Reply:
We seem to prefer the stream prefix, leaving this as is as other relevant files have it too. We could revisit that at some point and change them all at once.